boards/arm/beagle_play_cc1352/Kconfig.defconfig

```diff
@@ -25,15 +25,4 @@ config PINCTRL_CC13XX_CC26XX
 
 endif # PINCTRL
 
-if NETWORKING
-
-CONFIG_IEEE802154=y
-CONFIG_NET_PKT_TX_COUNT=6
-CONFIG_NET_BUF_DATA_SIZE=128
-CONFIG_IEEE802154_CC13XX_CC26XX=y
-CONFIG_IEEE802154_CC13XX_CC26XX_SUB_GHZ=y
-CONFIG_WPANUSB_DEVICE_VALID_CHANNELS=0x07FFFFFF
-
```

Comment:
You are cleaning up a file added in this PR, please squash the cleanup patch into the previous as it is not relevant to git history. Please add the CAPARRAY DELTA changes as part of this PR or https://github.com/zephyrproject-rtos/zephyr/pull/65048

Reply:
done